components/k8s-model-server/http-proxy/server.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import print_function
+
+from itertools import repeat
+import base64
+import logging
+
+from grpc.beta import implementations
+from tensorflow_serving.apis import predict_pb2
+from tensorflow_serving.apis import prediction_service_pb2
+from tornado import gen
+from tornado.ioloop import IOLoop
+from tornado.options import define, options, parse_command_line
+import tensorflow as tf
+import tornado.web
+
+
+define("port", default=8888, help="run on the given port", type=int)
+define("rpc_timeout", default=1.0, help="seconds for time out rpc request", type=float)
+define("rpc_port", default=9000, help="tf serving on the given port", type=int)
+define("rpc_address", default='localhost', help="tf serving on the given address", type=str)
+define("instances_key", default='instances', help="requested instances json object key")
+define("cookie_secret", default='YOU_NEED_THIS', help="cookie secrets")
```

Comment:
What is the cookie secret for?

Reply:
just a web server thing. if the web server is using cookie. maybe i should remove it. it is not required